Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,111 @@
+from ipaddress import IPv4Address, IPv4Network
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+DELIMITER = ","
+
+
+def is_ip_internal(ip: str, ranges: List[str]) -> bool:
+    try:
+        return any((IPv4Address(ip) in IPv4Network(cidr.split(DELIMITER)[0] if DELIMITER in cidr else cidr) for cidr in ranges))
+    except ValueError:
+        demisto.info("One or more IP ranges or IPs are invalid. Please make sure the list is in the correct structure.")
+        return True
```

Comment:
Don't you want the exception to be visible in the UI instead?

Reply:
Yes, this is a leftover from previous iteration of the code, fixing it